proto-wine/proton.cpp

```diff
@@ -58,14 +58,15 @@ std::tuple<QProcessEnvironment, QString, bool> make_steam_environ(const QString&
         error = QString("Couldn't find a Wineprefix for AppId %1").arg(appid);
 
     QString path = expand(
-        ":PROTON_PATH/dist/bin"
+        ":PROTON_DIST_PATH/bin"
+        ":PROTON_DIST_PATH/bin"
```

Comment:
Copy-paste error?

Reply:
Yup, leftover from a previous iteration where I was just creating duplicate paths for setting `LD_LIBRARY`/`PATH`, fixed.